### PR TITLE
BlogTruyen: Fix parse page list not return enough pages in some manga

### PR DESCRIPTION
**Pull request by VietAnh14**
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: src/vi/blogtruyen/src/eu/kanade/tachiyomi/extension/vi/blogtruyen/BlogTruyen.kt [hunk 1]
```diff
@@ -14,6 +14,7 @@ import okhttp3.HttpUrl.Companion.toHttpUrlOrNull
 import okhttp3.OkHttpClient
 import okhttp3.Request
 import okhttp3.Response
+import org.json.JSONArray
```
Comment (alessandrojean) [inline src/vi/blogtruyen/src/eu/kanade/tachiyomi/extension/vi/blogtruyen/BlogTruyen.kt:17]: Why not use `kotlinx.serialization` like the other extensions?

Comment (VietAnh14) [reply src/vi/blogtruyen/src/eu/kanade/tachiyomi/extension/vi/blogtruyen/BlogTruyen.kt:17]: I haven't looked at other extensions and get used to use `JSONArray`, I will change to use `kotlinx.serialization` as suggested.

Comment (alessandrojean) [reply src/vi/blogtruyen/src/eu/kanade/tachiyomi/extension/vi/blogtruyen/BlogTruyen.kt:17]: It's just a suggestion, we didn't remove the org.json usages in some extensions yet. You can let as it is if you want :+1:

**Review by alessandrojean — APPROVED** [review]

File: src/vi/blogtruyen/src/eu/kanade/tachiyomi/extension/vi/blogtruyen/BlogTruyen.kt (hunk 1 above)
Comment (VietAnh14) [reply src/vi/blogtruyen/src/eu/kanade/tachiyomi/extension/vi/blogtruyen/BlogTruyen.kt]: Thank you, but I will just use it as suggested to make it consistent with other extensions.

**Pull request merged**